client/post-editor/editor-diff-viewer/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getPostRevision,
+	normalizeForDisplay,
+} from 'state/posts/revisions/selectors';
+
+class EditorDiffViewer extends PureComponent {
+	render() {
+		return (
+			<div className="editor-diff-viewer">
+				<h3 className="editor-diff-viewer__title">
```

Comment:
No real reasoning :) Updated to a `h1` since it can be considered as the "main" title of the page when the diff viewer is shown?